x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/index.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IRouter } from 'kibana/server';
+import { GetTrustedAppsRequestSchema } from '../../../../common/endpoint/schema/trusted_apps';
+import { TRUSTED_APPS_LIST_API } from '../../../../common/endpoint/constants';
+import { getTrustedAppsListRouteHandler } from './handlers';
+import { EndpointAppContext } from '../../types';
+
+export const registerTrustedAppsRoutes = (
+  router: IRouter,
+  endpointAppContext: EndpointAppContext
+) => {
+  // GET list
+  router.get(
+    {
+      path: TRUSTED_APPS_LIST_API,
```

Comment:
I haven't seen other server code creating such constants.

Reply:
I wish they did. Both the UI and the Server can benefit from such constants since both need it. Ingest uses this heavily and it helps allot (ex. we import some of their consts and use it in our API calls - thus if they change, we will not break).

This const should be re-used by the UI's middleware when making the API call.

Reply:
@efreeti don't treat the existing endpoint/management server code as stylistic gospel. It was developed while we were still new to convention, and right after basic POCs. 

An argument can be made to "match what's there" not because it's _better_ but just for the sake of consistency. I can accept that point. Then if we want to change the style for all, it's an easier task if they all have the same pattern to -> from. 

However I also like and prefer this idea. We should just have a tech debt item to change the existing patterns to have const routes like this. It can then be quite easy to skim our defined URL endpoints in one place.